src/Redistribution.sol

```diff
@@ -697,19 +781,40 @@ contract Redistribution is AccessControl, Pausable {
      * @notice Returns true if current block is during reveal phase.
      */
     function currentRoundReveals() public view returns (Reveal[] memory) {
-        require(currentPhaseClaim(), "not in claim phase");
+        if (!currentPhaseClaim()) {
+            revert NotClaimPhase();
+        }
         uint256 cr = currentRound();
-        require(cr == currentRevealRound, "round received no reveals");
+        if (cr != currentRevealRound) {
+            revert NoReveals();
+        }
+
         return currentReveals;
     }
 
+    /**
+     * @notice Returns minimum depth reveal has to have to participate in this round
+     */
+    function currentMinimumDepth() public view returns (uint8) {
+        // We are checking this in  reveal phase, in this phase have new commit round but
+        // for claim round it should be still the last time pot was claimed, which essentialy gives
+        // us how many rounds have been skipped. Because of the difference by default we substract
+        // 1 from the last winner depth
```

Comment:
last time we agreed on keeping the same depth for the next subsequent round and only deducting it after if there was a missed round. in this current form, as you wrote, the depth requirement immediately looser with 1.

Reply:
also, try to rephrase the text because it doesn't have the correct grammar form. It is just my feelings, I also use deepl.com for writing text sometimes.